app/views/first-time/eligibility/claim/file-upload.html

```diff
@@ -11,35 +11,45 @@
   <div class="grid-row">
     <div class="column-two-thirds">
       <h2 class="heading-large">
-        Please upload
+        Please upload your {{ fileUploadGuidingText[document].displayname }}
```

Comment:
as per UX, the different doc types need a title and guidance text (may be empty) based on doc type:

https://apvs-alpha-ux.herokuapp.com/beta/v3/visit-conf